### ISSUE TITLE
RGB Pin D3 identification?

### ISSUE THREAD

**Issue by skatardude10** [nhou7/qmk_firmware_amj40#1]
Hello!! Thank you for this port of QMK for the AMJ-40 v2!!

I am looking into adding RGB underlighting with this firmware when my board arrives... But I am not the most skilled when it comes to electronics tbh.  I feel I can figure out the software side / flashing just fine but hardware wise, I don't want to muck up my board soldering the wrong pins for the RGB underlighting.  

Do you have any references, diagrams, links or thoughts I could use to help me successfully install RGB underlighting to this PCB? I've read through the QMK firmware docs / RGB modding section, I know what RGB modules to use. From pictures I see online, it looks like the AMJ 40 V2.0 board has a ton of labels/indicators... Is Pin D3 just straight up labeled? Do you know what 3 other pins I'd need to solder the RGB modules to the board are other than D3? Do the three other pins control power and D3 for color? 

Basically, I'm going to keep doing research but any help you could provide would be GREATLY appreciated!

Thanks!

**Comment by skatardude10** [comment]
Got your answer, Thanks man!

https://www.reddit.com/r/MechanicalKeyboards/comments/6565qr/buying_amj40_back_in_stock_at_aliexpress/dg9etw1/
